src/polus/config/user_inputs.py

```diff
@@ -9,7 +9,7 @@
 elements    = ["C","H","O","N","P","S"]
 prop_itls   = ["i","q"]
 
-def read_atom_names(input_dir):
+def read_atom_names(input_dir: str) -> list:
```

Comment:
Is it possible to be more specific about the return type here?

Python type hints allow you to specify the data type that appears in a list, dictionary or tuple. For example, a list of strings could be specified with `list[str]`.